packages/fiori/src/FlexibleColumnLayout.ts

```diff
@@ -507,10 +483,7 @@ class FlexibleColumnLayout extends UI5Element {
 	* <br><br>
 	* <b>For example:</b> ["67%", "33%", 0], ["100%", 0, 0], ["25%", "50%", "25%"], etc,
 	* where the numbers represents the width of the start, middle and end columns.
-	* @readonly
-	* @type {array}
-	* @defaultvalue ["100%", 0, 0]
-	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.columnLayout
+	* @default ["100%", 0, 0]
```

Comment:
we depend on _columnLayout property which default value is undefined. change this one to undefined